packages/wasm-sdk/src/sdk.rs

```diff
@@ -64,13 +66,127 @@ impl DerefMut for WasmSdkBuilder {
 #[wasm_bindgen]
 impl WasmSdkBuilder {
     pub fn new_mainnet() -> Self {
-        let sdk_builder = SdkBuilder::new_mainnet().with_context_provider(WasmContext {});
+        // Mainnet addresses - these are placeholder addresses for now
+        // TODO: Replace with actual mainnet addresses when available
+        let mainnet_addresses = vec![
+            "http://seed-1.mainnet.networks.dash.org:1443"
+                .parse()
+                .unwrap(),
+            "http://seed-2.mainnet.networks.dash.org:1443"
+                .parse()
+                .unwrap(),
+            "http://seed-3.mainnet.networks.dash.org:1443"
+                .parse()
+                .unwrap(),
+            "http://seed-4.mainnet.networks.dash.org:1443"
+                .parse()
+                .unwrap(),
+        ];
+
+        let address_list = dash_sdk::sdk::AddressList::from_iter(mainnet_addresses);
+        let sdk_builder = SdkBuilder::new(address_list)
+            .with_network(dash_sdk::dpp::dashcore::Network::Dash)
+            .with_context_provider(WasmContext {});
 
         Self(sdk_builder)
     }
 
+    pub fn new_mainnet_trusted() -> Result<Self, JsError> {
```

Comment:
IMO we don't need that _trusted variant, we trusted context should just replace previous WasmContext. It's so much better :).